File: app/src/main/java/com/dominikgaller/alpinebooking/booking/core/inport/ConfirmTourBookingCommand.java
package com.dominikgaller.alpinebooking.booking.core.inport;

/**
 * Command carrying the input data for the ConfirmTourBooking use case.
 *
 * <p>Contains only primitive and standard-library types — no domain value objects.
 *
 * <p>SDD: See {@code documentation/use-cases/uc02-confirm-tour-booking.spec.md}, section 2.
 */
public record ConfirmTourBookingCommand(String bookingId) {
}

File: src/main/kotlin/com/example/service/booking/core/inport/command/ConfirmTourBookingCommand.kt
package com.example.service.booking.core.inport.command

/**
 * Input for the ConfirmTourBooking use case (UC02).
 *
 * SDD: see `documentation/use-cases/uc02-confirm-tour-booking.spec.md` § 2.
 */
data class ConfirmTourBookingCommand(
    val bookingId: String,
)
